js_modules/dagster-ui/packages/eslint-config/rules/__tests__/missing-graphql-variables-type.test.js

```diff
@@ -36,26 +34,14 @@ ruleTester.run('missing-graphql-variables', rule, {
       import { SomeQuery, SomeQueryVariables } from '../SomeQueryWithVariables';
       useQuery<SomeQuery, SomeQueryVariables>();
     `,
-    `
-      import { SomeQuery } from '../SomeQueryWithOutVariables';
-      useQuery<SomeQuery>();
-    `,
     `
       import { SomeMutation, SomeMutationVariables } from '../SomeMutationWithVariables';
       useMutation<SomeMutation, SomeMutationVariables>();
     `,
-    `
-      import { SomeMutation } from '../SomeMutationWithOutVariables';
-      useMutation<SomeMutation>();
-    `,
     `
       import { SomeSubscription, SomeSubscriptionVariables } from '../SomeSubscriptionWithVariables';
       useSubscription<SomeSubscription, SomeSubscriptionVariables>();
     `,
-    `
-      import { SomeSubscription } from '../SomeSubscriptionWithOutVariables';
-      useSubscription<SomeSubscription>();
-    `,
```

Comment:
This was actually intentional, for the case where a query/mutation does not generate a variables type https://github.com/dagster-io/dagster/blob/4f07f14c61b52c0c05498e8d6c369619fa0640d3/js_modules/dagster-ui/packages/eslint-config/rules/__tests__/missing-graphql-variables-type.test.js#L21.

It was a bit defensive since I wasn't sure if that is ever the case. Maybe we don't need it because we always output a Variables type?

Reply:
https://github.com/dagster-io/internal/blob/aef30a88d4122a829d9ffde8385c8b7f4491e7b4/dagster-cloud/js_modules/app-cloud/src/app/types/CloudProvider.types.ts#L5

ah yeah we always output variables so its fine to delete this.